### PR TITLE
Bit update

### PR DESCRIPTION
**Pull request by burakukula**
**WILL BE MERGED 7.09.2018**
This is the first part of the update of text, that will be made in next days.
 - new config for font size and line-height - in case of BIT they are equal, so only one cofing needed.
 - custom function to getSizes
 - I will rename all the "context" like namespaces - "alt" "warning" and change them into color indicator modifiers "--blue-secondary" and "--peach-primary"
 - I am removing BIT behavior and "not-responsive" modifier. They should have the same simple meaning as other components. In projects I will replace each usage of BIT with according 2 components.

typography-handoff-map

### CODE REVIEW

File: src/sass/_config.scss [hunk 1]
```diff
@@ -80,6 +80,13 @@ $fontSizes: (
   header: 3rem
 );
 
+$bitSizes: (
```
Comment (aju) [inline src/sass/_config.scss]: I would keep it in bit scss ;)

File: src/sass/_mixins.scss [hunk 2]
```diff
@@ -42,6 +42,11 @@
   @return map-get($fontSizes, $fontsizeKey);
 }
 
+// $bitSizes one config for font size and line-height
```
Comment (aju) [inline src/sass/_mixins.scss]: I would keep it in bit scss ;)

File: src/sass/_config.scss (hunk 1 above)
Comment (burakukula) [reply src/sass/_config.scss]: @aju why? I think would be better to keep config for all types of texts in one place?

File: src/sass/_mixins.scss (hunk 2 above)
Comment (burakukula) [reply src/sass/_mixins.scss]: @aju ok, I will move config and function into BIT scss. You have a point, we will keep all the styles related to the component in it's scss.

**Pull request merged**